modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -681,4 +681,163 @@
 			font-size: 14px;
 		}
 	}
+}
+
+.col-md-4 {
+	.calendar-portlet-column-parent {
+		.calendar-list-item-arrow {
+			font-size: 1.75em;
+			top: 4px;
+
+			.icon-caret-down {
+				padding: 0 6px;
+			}
+		}
+
+		.calendar-portlet-column-grid, .calendar-portlet-column-options {
+			width: 100%;
+		}
+
+		.calendar-portlet-list-header {
+			.calendar-list-item-arrow {
+				top: 6px;
+			}
+		}
+
+		.scheduler-base-content .scheduler-base-hd .scheduler-base-views {
+			margin-top: 4px;
+		}
+
+		.scheduler-base-content .scheduler-base-view-date,
+		.scheduler-base-content .scheduler-base-controls,
+		.scheduler-base-content .scheduler-base-controls .btn-group,
+		.scheduler-base-content .scheduler-base-hd .scheduler-base-views {
+			text-align: center;
+			width: 100%;
+		}
+
+		.scheduler-base-controls {
+			.btn-group .calendar-add-event-btn {
+				margin-bottom: 5px;
+				width: 100%;
+			}
+		}
+
+		.scheduler-base-controls .btn-group .scheduler-base-today {
+			width: 50%;
+		}
+
+		.scheduler-base-hd .btn-group button {
+			width: 25%;
+		}
+
+		.scheduler-view-day-header-day-weekday {
+			font-size: 0;
+
+			&:first-letter {
+				font-size: 14px;
+			}
+		}
+
+		.scheduler-view-table-header-day div {
+			overflow: hidden
+		}
+
+		.simple-color-picker-item {
+			height: 18px;
+			width: 18px;
+		}
+	}
+}
+
+.col-md-6, .col-md-8 {
```

Comment:
`col-md-6` fixes the styling in a 50 column, and `col-md-8` fixes stying in a 70 column.